superset-frontend/cypress-base/package.json

```diff
@@ -10,14 +10,15 @@
   "author": "Apcahe",
   "license": "Apache-2.0",
   "dependencies": {
-    "@cypress/code-coverage": "^3.9.2",
+    "@cypress/code-coverage": "^3.9.11",
     "@superset-ui/core": "^0.17.42",
     "react-dom": "^16.13.0",
     "rison": "^0.1.1",
     "shortid": "^2.2.15"
   },
   "devDependencies": {
-    "cypress": "^6.3.0",
+    "@cypress/webpack-dev-server": "^1.6.0",
+    "cypress": "^6.9.1",
```

Comment:
should this be 7x?

Reply:
Whoops, I did everything except the most important thing! 😅 
Fixed in [`this commit`](https://github.com/apache/superset/pull/16734/commits/5c1fc6feccfa3e0fbb1b6306be1964793780586c).